docs/en/sql-reference/table-functions/deltalake.md

```diff
@@ -6,25 +6,24 @@ sidebar_label: deltaLake
 
 # deltaLake Table Function
 
-Provides a read-only table-like interface to [Delta Lake](https://github.com/delta-io/delta) tables in Amazon S3.
+Provides a read-only table-like interface to [Delta Lake](https://github.com/delta-io/delta) tables in Amazon S3 or Azure Blob Storage.
 
 ## Syntax
 
 ``` sql
-deltaLake(url [,aws_access_key_id, aws_secret_access_key] [,format] [,structure] [,compression])
+deltaLakeS3(url [,aws_access_key_id, aws_secret_access_key] [,format] [,structure] [,compression])
```

Comment:
Worth to leave example with just "deltaLake()" and mention that for compatibility it would read from s3.

Reply:
addressed it